aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -329,23 +329,17 @@ Route handlers are methods that execute when the route matches. Route handlers c
 
 [!code-csharp[](minimal-apis/samples/WebMinAPIs/Program.cs?name=snippet_sm)]
 
-### Name routes and link generation
+### Name endpoints and link generation
 
-Routes can be given names in order to generate URLs to the route. Using a named route avoids having to hard code paths in an app:
+Endpoints can be given names in order to generate URLs to the endpoint. Using a named endpoint avoids having to hard code paths in an app:
```

Comment:
> Endpoints can be given names in order to generate URLs to the endpoint.

This sentence feels a little confusing. What exactly are we trying to say here?

Reply:
That endpoint names provide some indirection when generating links? I didn't write this originally, so I'm guessing. Do you have a suggestion for how we should reword this?

Reply:
Providing the readers with a lot of examples makes it much clearer.  :-)

```csharp
var builder = WebApplication.CreateBuilder(args);

var app = builder.Build();

var users = new List<User>
 {
    new User(1,"A"),
    new User(2,"B"),
    new User(3,"C"),
 };

IResult GetUser(int id)
{
    return users.Find(u => u.id == id) is User user ?
        TypedResults.Ok(user) : TypedResults.NotFound();
}

IResult CreateUser(User user, LinkGenerator lg)
{
    users.Add(user);
    var path = lg.GetPathByName(nameof(GetUser), new { id = user.id })!;
    return TypedResults.Created(path, user);
}

app
    .MapGet("/users/{id}", GetUser)
    .WithName(nameof(GetUser));

app.MapPost("/users", CreateUser);

app.Run();

record class User(int id, string name);
```